src/styles/annotator/components/sidebar.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * Styles for the container and iframe containing the sidebar app. These
+ * classes are used by the `Sidebar` class (not a UI component).
+ */
+
+@layer utilities {
```

Comment:
This is just a tiny layer of abstraction to group these together and comment them. One of these classes is used twice, one just once.